upstream/solr6/local/92-init-solr.sh

```diff
@@ -269,7 +269,7 @@ makeConfigs
 
 user="solr"
 # make sure backup folders exist and have the right permissions in case of mounts
-for coreName in "${DEFAULT_CORES[@]}"
+for coreName in "${DEFAULT_CORES_ALFRESCO[@]}"
```

Comment:
This does mean we dont make backup dirs for archive, correct?
Seeing as the archive core is not that important for us, this might be fine?

Reply:
Good point. I've added changed my approach. Now using a function to create the backup dir instead of looping over the default cores.